modules/code_metrics.js

```diff
@@ -0,0 +1,20 @@
+const jshint = require('../lib/jshint-2.11.1/jshint.js')
+
+/**
+ * Performs static analysis on code 
+ * @param {string} code source code 
+ * @return {Object} metrics
+ */
```

Comment:
I think you meant `**/` rather than `*/`?

Reply:
`/** */` is the formatting for jsdoc!